athena_cli.py

```diff
@@ -22,39 +23,54 @@
 
 __version__ = '0.1.8-bt'
 
-def output_results(athena, format, execution_id, output):
-    results = athena.get_query_results(execution_id)
-    headers = [h['Name'].encode("utf-8") for h in results['ResultSet']['ResultSetMetadata']['ColumnInfo']]
+def output_results(athena, format, execution_id, output, is_shell):
+    is_csv = format in ['CSV', 'CSV_HEADER', 'TSV', 'TSV_HEADER']
 
-    row_count = len(results['ResultSet']['Rows'])
-    if headers and len(results['ResultSet']['Rows']) and results['ResultSet']['Rows'][0]['Data'][0].get('VarCharValue', None) == headers[0]:
-        row_count -= 1  # don't count header
+    #tabulate() only supports ascii
```

Comment:
Is this true for Python 3?

Reply:
It seems to not be an issue in Python 3, I refactored this a little though and removed the comment

Reply:
I was going to suggest `athena` CLI completely drop support for Python 2.7 because cmd2, it's major dependency, has dropped support as of the end of last month. https://github.com/python-cmd2/cmd2#python-27-support-is-eol

So the charset problem might be resolved by just reverting the earlier hack. See #34

Reply:
We're using Python 3, so it's fine by us. I'll update the PR with some encoding cleanup and put in a python 3 shebang

Reply:
I went ahead and made the changes, let me know what you think.